pyaerocom/web/test/test_aerocom_evaluation.py

```diff
@@ -0,0 +1,181 @@
+import os
+import pytest
+
+from pyaerocom import Colocator, ColocatedData, GriddedData, UngriddedData
+from pyaerocom.web import AerocomEvaluation
+from pyaerocom.io.aux_read_cubes import add_cubes
+
+PROJ_ID = 'project'
+EXP_ID = 'exp'
+
+OBS_ID = 'AeronetSunV3L2Subset.daily'
+OBS_NAME = 'AeronetSun'
+OBS_VARS = 'od550aer'
+OBS_VERT_TYPE = 'Column'
+
+MODEL_NAME = 'TM5'
+MODEL_ID = 'TM5-met2010_CTRL-TEST'
+TS_TYPE = 'monthly'
+START = 2010
+
+@pytest.fixture(scope='function')
+def model_config():
+    config = {MODEL_NAME : dict(model_id=MODEL_ID,
+                                model_ts_type_read=TS_TYPE,
+                                vert_which=OBS_VERT_TYPE
+                                )}
+    return config
+
+@pytest.fixture(scope='function')
+def obs_config():
+    config = {OBS_NAME : dict(obs_id=OBS_ID, obs_vars=OBS_VARS, obs_vert_type=OBS_VERT_TYPE)}
+    return config
+
+@pytest.fixture(scope='function')
+def model_config_aux(model_config):
+    config = model_config.copy()
+    config[MODEL_NAME]['model_read_aux'] = {'od550aer' : dict(
+        vars_required=['od550aer', 'od550aer'],
+        fun='add_cubes')}
+    return config
+
+@pytest.fixture(scope='function')
+def stp(model_config, obs_config):
+    return AerocomEvaluation(proj_id=PROJ_ID, exp_id=EXP_ID, model_config=model_config,
+                             obs_config=obs_config, start=START, ts_type=TS_TYPE,
+                             raise_exceptions=True, reanalyse_existing=True)
+
+@pytest.fixture(scope='function')
+def stp_min():
+    return AerocomEvaluation(proj_id=PROJ_ID, exp_id=EXP_ID, reanalyse_existing=True,
+                             raise_exceptions=True)
+
+
+def test_aerocom_evaluation(stp_min):
+    assert isinstance(stp_min, AerocomEvaluation)
+
+def test_aerocom_evaluation_run_colocation(stp):
+    col = stp.run_colocation(model_name='TM5', obs_name='AeronetSun', var_name='od550aer')
+    assert isinstance(col, Colocator)
+
+    col_paths = stp.run_evaluation(update_interface=False)
+    assert len(col_paths) == 1
+    assert os.path.isfile(col_paths[0])
+
+def test_aerocom_evaluation_update_heatmap_json(stp, tmpdir):
+    stp.run_evaluation(update_interface=False)
+    stp.update_heatmap_json()
+
+def test_aerocom_evaluation_get_web_overview_table(stp, tmpdir):
+    stp.update()
+    stp.run_evaluation(update_interface=False)
+    table = stp.get_web_overview_table()
+    from pandas import DataFrame
+    assert isinstance(table, DataFrame)
+    assert len(table) > 0
+
+def test_aerocom_evaluation_get_custom_read_method_model_file(stp, model_config_aux):
+    methods_file = '/home/eirikg/Desktop/pyaerocom/data_new/eval_py/cube_read_methods.py'
+    stp.add_methods_file = methods_file
+    stp.model_config = model_config_aux
+    fun = stp.get_custom_read_method_model('add_cubes')
+    assert fun == add_cubes
```

Comment:
@jgliss: Do you have a suggestion on how to deal with needing this file for testing? Here I just copied it from the data_new repo..
Should we add it to the testdata? Create it "on the fly"?